README.md

```diff
@@ -12,7 +12,7 @@
 
 ## License
 
-This is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
+This SAFE Network library is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
```

Comment:
@pravarseth , shall we consider changing it for cases like this one which is not a library but a SAFE Network application?

Reply:
@bochaco so change it to: "This SAFE Network `application` is dual-licensed.."?

Reply:
yes @nadiaburborough , I think it will be more accurate for sure.